### ISSUE TITLE
Allow use of C# projects using Refit to be referenced VB.Net Projects

### ISSUE THREAD

**Issue by schallm** [reactiveui/refit#735]
We are writing a new web service in C# that will be called by other projects (some written in VB.Net).  We have written a client dll (C#) for the web service (C#) for other projects to use to consume the web service.  The client dll (C#) uses Refit to talk to the web services.

When adding a reference to the client dll (C#) within a VB.Net project and building, we receive errors from a RefitStubs.j.cs file.

I know from other tickets, that Refit doesn't currently support VB.Net (or other languages) projects.  However, is there a way to use libraries written in C# that use Refit within a VB.Net (or other languages) projects?  Can the generation maybe be done when compiling the C# library?

Below is the "file" that is causing the issues after the reference to my client dll (C#) was added to a VB.Net project.

```cs
// <auto-generated />
using System;
using System.Net.Http;
using System.Collections.Generic;

/* ******** Hey You! *********
 *
 * This is a generated file, and gets rewritten every time you build the
 * project. If you want to edit it, you need to edit the mustache template
 * in the Refit package */

#pragma warning disable
namespace RefitInternalGenerated
{
    [global::System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
    [AttributeUsage (AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Event | AttributeTargets.Interface | AttributeTargets.Delegate)]
    sealed class PreserveAttribute : Attribute
    {

        //
        // Fields
        //
        public bool AllMembers;

        public bool Conditional;
    }
}
#pragma warning restore
```

**Comment by benjaminhowarth1** [comment]
Odd. I'll check this out.

**Comment by benjaminhowarth1** [comment]
@schallm apologies for the delay in picking this up. If you create your interfaces in a separate C# project, as far as I know, there's no reason why you can't then reference that C# project in a VB.NET project. Can you show me a rough sample of what you're working with?

**Comment by benjaminhowarth1** [comment]
I'm going to close this as I merged a PR that fixes #811 for an F# project, and therefore should also fix this issue too. If it requires re-opening, let me know!
